erpnext/accounts/doctype/sales_invoice_item/sales_invoice_item.json

```diff
@@ -455,7 +455,6 @@
    "print_hide": 1
   },
   {
-   "default": ":Company",
```

Comment:
why was this removed?

Reply:
In sales invoice, When I select another company than the default company & add the item in the item table,
because of this default company value for the cost center. It always returns the default company's cost center for the item.
& it throwing an error while submitting the sales invoice.